e2e/pages/adf/contentServicesPage.ts

```diff
@@ -61,16 +61,24 @@ export class ContentServicesPage {
     copyButton = element(by.css('button[data-automation-id="content-node-selector-actions-choose"]'));
     searchInputElement = element(by.css('input[data-automation-id="content-node-selector-search-input"]'));
     shareNodeButton = element(by.cssContainingText('mat-icon', ' share '));
+    nameColumn = by.css('div[class*="datatable-body"] div[class*="adf-data-table-cell"][title="Display name"]');
```

Comment:
We shouldn't have locators with datatable-body on this page. We should just provide the name of the title in this case('Display name') to the documentListPage, which will provide to datatablePage that should have a general locator: by.css('div ... [title="' + titleName + "]') and where the titleName will be provided from contentServicesPage.